pkg/util/common.go

```diff
@@ -95,6 +99,80 @@ func Replace(table map[string]string, s string) string {
 	return s
 }
 
+// DynamicReplace 动态替换字符串中的变量
+func DynamicReplace(regPattern string, fsSeparator string, rule string, pathAvailable bool, thumbAvailable bool,
```

Comment:
Please remove `regPattern` from the parameter list, since it's static across all references.